Gopkg.lock

```diff
@@ -2,12 +2,15 @@
 
```

Comment:
We should probably merge the other PR that changes Gopkg.lock before this one.

Reply:
Is that taken care of now? Not sure which story you were thinking of.